bundles/core/pom.xml

```diff
@@ -462,6 +462,12 @@
             <scope>test</scope>
         </dependency>
         <!-- for testing we need the new ResourceTypeBasedResourcePicker -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.32</version>
+            <scope>tes</scope>
```

Comment:
Scope should be `test`

Reply:
Good catch, fixed in https://github.com/adobe/aem-core-email-components/commit/3137c8c25f331c12424b7bccd300c75d818612d1 on development.